app/krs.js

```diff
@@ -124,6 +124,13 @@ exports.provisionKey = co(function *(req) {
     throw utils.ErrorResponse(400, 'email required');
   }
 
+  const krsSpecific = req.body.krsSpecific;
+  if (krsSpecific) {
+    
+    // Optional ToDo for KRS provider: implement custom code to handle the krsSpecific parameter
```

Comment:
Can you provide an example of what a KRS provider would add here? Is this simply just a hook to allow different KRS providers to execute custom logic when provisioning keys?

Reply:
Why would they need to execute custom logic here, as opposed to after the key has been created?

Reply:
yes, this is basically a hook that allows the platform and user to pass in extra information to the krs provider, information that may be specific to the particular provider.
we thought DAI was going to need this because they have different offerings (insurance, etc), but it turns out they don't. we could still go forward and add this in case somebody wants it in the future.